[DOCS] Gtm/cta in docs

bhcastleton opened the pull request:
[DOCS]
Changes proposed in this pull request:
This PR adds two links and associated text in the documentation to help people find Great Expectations Cloud if they're looking for a saas solution. One at the bottom of the welcome page, and another after the tutorial at the end of the customize your deployment section.

bhcastleton committed:
Update intro.md

Adding a CTA to sign up for our cloud workshops at the bottom of the welcome Documentation.

bhcastleton committed:
Update customize_your_deployment.md

Adding a gtm CTA in the customize your deployment documentation.

Rachel-Reverie commented on docs/intro.md:
```diff
@@ -101,4 +101,7 @@ Slack is good for that, too: [join slack](https://greatexpectations.io/slack).
+We welcome contributions to Great Expectations. Please start with our [Contributing guide](/docs/contributing/contributing) and don’t be shy with questions!
+
+#### Not interested in managing your own configuration or infrastructure? 
+Learn more about Great Expectations Cloud — our fully managed SaaS offering. Sign up for our weekly cloud workshop [here](https://greatexpectations.io/cloud). You’ll get to see our newest features and apply for our private Alpha program!
```
```suggestion
Learn more about Great Expectations Cloud — our fully managed SaaS offering. Sign up for [our weekly cloud workshop](https://greatexpectations.io/cloud)! You’ll get to see our newest features and apply for our private Alpha program!
```

Rachel-Reverie replied:
In general it is preferred to have a link to a thing or action be the description of the thing or action, rather than having the thing or action be described and then providing a generic "here" link.

talagluck reviewed (approved):
Looks good to me!!!

talagluck committed:
Merge branch 'develop' into gtm/cta_in_docs

Rachel-Reverie commented on docs/tutorials/getting_started/customize_your_deployment.md:
```diff
@@ -97,3 +97,6 @@ Most teams will want to configure various Checkpoints and Validation Actions as
 * [How to run a Checkpoint in Airflow](/docs/deployment_patterns/how_to_run_a_checkpoint_in_airflow)
+
+## Not interested in managing your own configuration or infrastructure?
+Learn more about Great Expectations Cloud — our fully managed SaaS offering. Sign up for our weekly cloud workshop [here](https://greatexpectations.io/cloud). You’ll get to see our newest features and apply for our private Alpha program!
```
```suggestion
Learn more about Great Expectations Cloud — our fully managed SaaS offering. Sign up for [our weekly cloud workshop](https://greatexpectations.io/cloud)! You’ll get to see our newest features and apply for our private Alpha program!
```

Rachel-Reverie replied:
Same change as above, for the same reason.

bhcastleton committed:
Update docs/intro.md

Making the link be the description of the link instead of the word "here."

Co-authored-by: Rachel-Reverie <94694058+Rachel-Reverie@users.noreply.github.com>

bhcastleton committed:
Update docs/tutorials/getting_started/customize_your_deployment.md

making the link be the description of the linked page instead of the word "here."

Co-authored-by: Rachel-Reverie <94694058+Rachel-Reverie@users.noreply.github.com>

abegong reviewed (approved):
❤️ ❤️ ❤️

bhcastleton committed:
Merge branch 'develop' into gtm/cta_in_docs

Rachel-Reverie reviewed (approved):
Looks good to me.
